lib/Db/FeedMapperV2.php

```diff
@@ -186,21 +186,28 @@ function ($value): int {
             },
             $this->db->executeQuery($idBuilder->getSQL(), $idBuilder->getParameters())->fetchAll()
         );
-        $time = new Time();
-        $builder = $this->db->getQueryBuilder();
-        $builder->update(ItemMapperV2::TABLE_NAME)
-            ->set('unread', $builder->createParameter('unread'))
-            ->set('last_modified', $builder->createParameter('last_modified'))
-            ->andWhere('id IN (:idList)')
-            ->andWhere('unread != :unread')
-            ->setParameter('unread', false, IQueryBuilder::PARAM_BOOL)
-            ->setParameter('idList', $idList, IQueryBuilder::PARAM_INT_ARRAY)
-            ->setParameter('last_modified', $time->getMicroTime(), IQueryBuilder::PARAM_STR);
-
-        return $this->db->executeStatement(
-            $builder->getSQL(),
-            $builder->getParameters(),
-            $builder->getParameterTypes()
-        );
+        
+		$chunked_idList = array_chunk($idList,65500,true);
```

Comment:
Any particular reason to preserve the keys?

Reply:
This was done "just in case" :)
Although I looked in more detail, there’s probably no point in saving the keys.
I will cancel saving the keys